src/WebJobs.Script/ScriptHostBuilderExtensions.cs

```diff
@@ -285,6 +288,7 @@ public static IHostBuilder AddScriptHostCore(this IHostBuilder builder, ScriptAp
                 if (!applicationHostOptions.HasParentScope)
                 {
                     AddCommonServices(services);
+                    services.AddAzureStorageProvider();
```

Comment:
Previously at this layer we took no dependencies on Azure Storage, the config model being that customers add services/extensions they need on startup.  Functions WebHost calls AddScriptHostCore to register these core services, and is the one that brings in the additional storage dependencies. We should be able to remove this and let WebHost add the storage dependencies?